CenterForOpenScience/ember-osf-web
# [EMB-370] User settings profile name

## Purpose

Allow user to change their name parts. Note that this is based on #465 and #469. Those should probably get merged first.

## Summary of Changes

1. Added names changing components
2. Added citation preview
3. Tests

## Feature Flags

Part of `ember_user_settings_profile_page`

## QA Notes

See ticket for acceptance tests which may be used to create your tests sessions.

## Ticket

https://openscience.atlassian.net/browse/EMB-370

# Reviewer Checklist
